sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -381,6 +382,7 @@ class DeviceBinaryImage {
   DeviceBinaryImage::PropertyRange DeviceLibReqMask;
   DeviceBinaryImage::PropertyRange KernelParamOptInfo;
   DeviceBinaryImage::PropertyRange ProgramMetadata;
+  DeviceBinaryImage::PropertyRange ExportedSymbols;
```

Comment:
@smaslov-intel do we have usages of this structure outside of `sycl/source` directory? If not, we should move this class inside the lib, otherwise this is a breaking change.

Reply:
Good point. I see something similar is done for `getSpecConstantsDefaultValues`. I will adopt that approach.

Reply:
It has been changed. It should no longer be a breaking change. Thanks for pointing it out!